packages/docs/content/blog/index.mdx

```diff
@@ -0,0 +1,94 @@
+
+#### Quality-assurance improvements
+
+We made a few big changes to our automated testing tools. 
+
+- Migrated our automated accessibility tests (axe-core) away from testing our legacy doc-site examples to testing our much larger suite of Storybook stories. 
+- Added a11y tests to our continuous-integration job. That helps us validate all our code changes for accessibility as we make them. 
+- Finished converting all unit tests to use [react-testing-library](https://testing-library.com/docs/react-testing-library/intro/). While the goal of this was to make them compatible with React 17, the process also resulted in higher-quality tests. 
+- Rebuilt our visual-regression-testing suite to test all three themes of our design system on multiple browsers at multiple screen sizes. 
+
+Mamma mia, those are some thorough checks! All these things help us be more confident that we're providing quality components.
+
+#### Sketch alert symbol update 
+
+We have consolidated and updated our alert symbol to take it from multiple alert symbols to a single alert symbol with optional overrides. The override panel supports standard and lightweight designs, changing the alert icon, and showing and hiding the alert heading and body. The simplified approach to the alert and the practice of utilizing layer styles more to achieve design flexibility is something we are going to continue weaving into other sketch symbols as we create and update them. Finally, this updated alert symbol now can easily adapt to small screen sizes without breaking the design of the alert.
+
+<div className="ds-u-measure--wide ds-u-margin-top--2">
+  <Video name="sketch-alert-demo" />
+</div>
+
+#### Camilo Sketch plug-in discovery 
+[We have been researching options](https://github.com/CMSgov/design-system/discussions/2231) for making our Sketch UI-kits easier to maintain and scale. Currently, we maintain three UI-kits for Healthcare, Medicare, and the Core Design System. Each time there is a change needed we have to make that change to the three different UI-kits. This can be time consuming and creates challenges when trying to replicate the exact symbol in different Sketch UI-kits. One of the more promising solutions to solve this problem is a plugin for Sketch called Camilo. It applies a theme to a single UI-kit so you can have multiple brands use the same UI-kit. The Camilo plugin has some issues that prevent it from being a solid solution for the CMS Design System. We are continuing to do some final explorations before closing the door on this option.
```

Comment:
```suggestion
[We have been researching options](https://github.com/CMSgov/design-system/discussions/2231) for making our Sketch UI kits easier to maintain and scale. Currently, we maintain three UI kits for Healthcare, Medicare, and the Core Design System. Each time there is a change needed we have to make that change to the three different UI kits. This can be time-consuming and creates challenges when trying to replicate the exact symbol in different Sketch UI kits. One of the more promising solutions to solve this problem is a plugin for Sketch called Camilo. It applies a theme to a single UI kit so you can have multiple brands use the same UI kit. The Camilo plugin has some issues that prevent it from being a solid solution for the CMS Design System. We are continuing to do some final explorations before closing the door on this option.
```